arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -2667,6 +2667,10 @@
           "type": "boolean",
           "description": "Gets or sets whether the extension handler should be automatically upgraded across minor versions."
         },
+        "forceUpdateTag": {
+          "type": "string",
+          "description": "Gets or sets whether the extension handler should be forced to re-run even if the extension configuration has not changed."
```

Comment:
The beginning phrase ```Gets or sets``` is a required for all sentences (for coding convention I think).